application/controllers/GroupController.php

```diff
@@ -225,23 +228,42 @@ public function addmemberAction()
         $this->assertPermission('config/access-control/groups');
         $groupName = $this->params->getRequired('group');
         $backend = $this->getUserGroupBackend($this->params->getRequired('backend'), 'Icinga\Data\Extensible');
-
-        $form = new AddMemberForm();
-        $form->setDataSource($this->fetchUsers())
+        $form = (new AddMemberForm())
             ->setBackend($backend)
             ->setGroupName($groupName)
             ->setRedirectUrl(
                 Url::fromPath('group/show', array('backend' => $backend->getName(), 'group' => $groupName))
```

Comment:
Why here still `Icinga\Web\Url` and not `IplUrl`? The `AddMemberForm` now isn't a `Icinga\Web\Form` anymore and so doesn't require it. It even asks for a string anyway, so we should at least pass a ipl compatible object here.